### DISCUSSION TITLE
Using jekyll but reading README instead of index.html

### DISCUSSION THREAD

**Discussion by rezy12** [community/community#102794] [Other Feature Feedback, Questions, & Ideas]
Hello I am new to react and was wondering why my website page shows the `read.me` instead of my `index.html` file. I created the page and was running completely fine on my computer but when I upload it on GitHub the `read.me` is only showing. I have already tried the switching of branches to `gh-pages` `settings` `pages branch to gh-pages root` save but it is still hosting the readme.md. So i then tried to use `jekyll` hoping that would work but still same problem. I tried to look as other discussion but still don't know the problem. 

here is the code: https://github.com/rezy12/BlisfulBites

can someone help me please because I tried everything that I know and do not understand what the problem is, Thank you in advanced.

**Comment by AndreaGriffiths11** [comment]
Hey @rezy12, in your repo it looks like your file `index.html` is in a subfolder called `public`, and not directly inside the root directory. This placement is important for GitHub Pages to recognize it as the main page of your site. I hope this helps!

**Reply by rezy12** [reply]
Thank you Andrea but I tried taking the file out of the public to the root but same problem unless I didn't do it right, again I'm new to all this. I also have a project similar to this but same layout but it works for that really confused why this is happening. Here is the photo of the layout:

![Screenshot 2024-01-30 163842](https://github.com/community/community/assets/45492208/256c43a6-d148-450a-a422-d3698bae9e98)
